### PR TITLE
l10n: uk: update translation

### PR DESCRIPTION
**Pull request by arkid15r**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -9909,29 +9912,29 @@ msgstr "не вдалося створити директорію \"%s\""
 
 #: builtin/log.c:2264
 msgid "--interdiff requires --cover-letter or single patch"
-msgstr ""
+msgstr "--interdiff вимагає --cover-letter або окремої латки"
 
 #: builtin/log.c:2268
 msgid "Interdiff:"
-msgstr ""
+msgstr "Різниця з попередньою серією латок:"
```
Comment (kasya) [inline po/uk.po:9919]: 👍🏼

File: po/uk.po [hunk 2]
```diff
@@ -9909,29 +9912,29 @@ msgstr "не вдалося створити директорію \"%s\""
...
 #: builtin/log.c:2269
 #, c-format
 msgid "Interdiff against v%d:"
-msgstr ""
+msgstr "Різниця з v%d:"
 
 #: builtin/log.c:2279
 msgid "--range-diff requires --cover-letter or single patch"
-msgstr ""
+msgstr "--range-diff вимагає --cover-letter або окремої латки"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "--range-diff потребує --cover-letter або окремої латки"
```

**Review by kasya — COMMENTED** [review]
LGTM!

File: po/uk.po (hunk 2 above)
Comment (arkid15r) [reply po/uk.po]: yep 👍

**Pull request merged**